src/shared/codelens/typescriptCodeLensProvider.ts

```diff
@@ -112,18 +116,13 @@ export async function makeTypescriptConfig(
 /**
  * Launches and attaches debugger to a SAM Node project.
  */
-export async function invokeTypescriptLambda(
-        ctx: ExtContext,
-        config: NodejsDebugConfiguration,
-        ) {
+export async function invokeTypescriptLambda(ctx: ExtContext, config: NodejsDebugConfiguration) {
     // Switch over to the output channel so the user has feedback that we're getting things ready
     ctx.chanLogger.channel.show(true)
     ctx.chanLogger.info('AWS.output.sam.local.start', 'Preparing to run {0} locally...', config.handlerName)
 
     const processInvoker = new DefaultValidatingSamCliProcessInvoker({})
-    config.samLocalInvokeCommand = new DefaultSamLocalInvokeCommand(ctx.chanLogger, [
-        WAIT_FOR_DEBUGGER_MESSAGES.NODEJS
-    ])
+    config.samLocalInvokeCommand = new DefaultSamLocalInvokeCommand(ctx.chanLogger, [WAIT_FOR_DEBUGGER_MESSAGES.NODEJS])
```

Comment:
what's doing this auto-formatting? do we have project-wide style settings somewhere? I don't see how this change is an improvement.

Reply:
Likely Husky. It's autorun on commit.